lib/pages/chat_details/participant_list_item/participant_list_item.dart

```diff
@@ -0,0 +1,218 @@
+
+  const ParticipantListItem(
+    this.member, {
+    Key? key,
+    this.onUpdatedMembers,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final membershipBatch = <Membership, String>{
+      Membership.join: '',
+      Membership.ban: L10n.of(context)!.banned,
+      Membership.invite: L10n.of(context)!.invited,
+      Membership.leave: L10n.of(context)!.leftTheChat,
+    };
+    final permissionBatch = member.powerLevel == 100
+        ? L10n.of(context)!.admin
+        : member.powerLevel >= 50
+            ? L10n.of(context)!.moderator
+            : '';
+
+    return Opacity(
+      opacity: member.membership == Membership.join ? 1 : 0.5,
+      child: ListTile(
+        onTap: () async => await _onItemTap(context),
+        title: Row(
+          children: <Widget>[
+            Flexible(
+              child: Text(
+                member.calcDisplayname(),
+                overflow: TextOverflow.ellipsis,
+                maxLines: 1,
+              ),
+            ),
+            if (permissionBatch.isNotEmpty)
+              Container(
+                padding: ParticipantListItemStyle.permissionBatchTextPadding,
+                margin: ParticipantListItemStyle.permissionBatchMargin,
+                decoration: BoxDecoration(
+                  color: Theme.of(context).colorScheme.primaryContainer,
+                  borderRadius: ParticipantListItemStyle.permissionBatchRadius,
+                  border: Border.all(
+                    color: Theme.of(context).colorScheme.primary,
+                  ),
+                ),
+                child: Text(
+                  permissionBatch,
+                  style: TextStyle(
+                    fontSize:
+                        ParticipantListItemStyle.permissionBatchTextFontSize,
+                    color: Theme.of(context).colorScheme.primary,
+                  ),
+                ),
+              ),
+            membershipBatch[member.membership]!.isEmpty
+                ? Container()
+                : Container(
+                    padding: ParticipantListItemStyle.membershipBatchPadding,
+                    margin: ParticipantListItemStyle.membershipBatchMargin,
+                    decoration: BoxDecoration(
+                      color: Theme.of(context).secondaryHeaderColor,
+                      borderRadius:
+                          ParticipantListItemStyle.membershipBatchRadius,
+                    ),
+                    child: Center(
+                      child: Text(membershipBatch[member.membership]!),
+                    ),
+                  ),
+          ],
+        ),
+        subtitle: Text(member.id),
+        leading:
+            Avatar(mxContent: member.avatarUrl, name: member.calcDisplayname()),
+      ),
+    );
+  }
+
+  Future _onItemTap(BuildContext context) async {
+    final responsive = getIt.get<ResponsiveUtils>();
+
+    if (PlatformInfos.isMobile || responsive.isMobile(context)) {
```

Comment:
i think `responsive.isMobile(context)` is  enough. because if the device is tablet, PlatformInfos.isMObile is true

Reply:
fixed: https://github.com/linagora/twake-on-matrix/pull/1596/commits/166cd8a7307d82ff9e81c4ecae07e34e60232cfe